### PR TITLE
Updating AirSwap and adding Rinkeby space

### PR DESCRIPTION
**Pull request by dmosites**

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
- It's now possible to create and edit a space without doing pull requests but using ENS and IPFS instead. Could you use this new method to create your space? Here is the guide: https://docs.snapshot.page/guides/create-a-space (you can keep your PR for add the images and skin)
- Could you create a PR to edit Airswap space and create the new one with ENS?
- You also should migrate your Airswap space on ENS if possible! Thank you.

**Comment by dmosites** [comment]
Thanks @gokaiorg I've configured ENS content records for `vote.airswap.eth` and `vote-rinkeby.airswap.eth` and updated the entries in `domains.json` and `aliases.json`. Let me know how it looks thank you!

File: spaces/aliases.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "aavegotchi": "aavegotchi.eth",
+  "airswap": "vote.airswap.eth",
+  "airswap-rinkeby": "vote-rinkeby.airswap.eth",
```
Comment (gokaiorg) [inline spaces/aliases.json:4]: Add a line here only if you have an existing space to migrate, airswap-rinkeby should not be there.
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
- @dmosites When you create a space with ENS, you should have an url looks like https://snapshot.page/#/vote.airswap.eth
- Please follow this doc to get more info to complete it https://docs.snapshot.page/guides/create-a-space
- Your space directory should be named `spaces/vote.airswap.eth/logo.png` instead of `spaces/airswap/logo.png` in this case.

**Comment by dmosites** [comment]
Thanks @gokaiorg should be good to go. I've reviewed the documentation and think we've got it right.

**Review by gokaiorg — APPROVED** [review]
@dmosites Everything looks good to me :) 

Just do you know why your https://snapshot.page/#/vote.airswap.eth and https://snapshot.page/#/vote-rinkeby.airswap.eth spaces doesn’t loads?

Have you filled out all the required fields and added a strategy?

**Comment by dmosites** [comment]
Awesome. Yup those have been updated, I'll separately request the existing proposals to be migrated over on Discord.

Thanks @gokaiorg!

**Comment by gokaiorg** [comment]
> Awesome. Yup those have been updated, I'll separately request the existing proposals to be migrated over on Discord.
> 
> Thanks @gokaiorg!

Great @dmosites! The migrated space https://snapshot.page/#/vote.airswap.eth doesn’t look to be loaded. Could you have a look on it before we merge an do the migration process? Thank you 😊

**Pull request merged**